<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="12" column="2">
     <widget class="QPushButton" name="load_btn">
      <property name="text">
       <string>load</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>rotate z:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>move z:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="scale_le">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QPushButton" name="set_btn">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>settings</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>scale:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="rot_y_le">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="move_x_le">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="move_z_le">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="14">
     <widget class="Viewer" name="viewer" native="true"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>move y:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="rot_z_le">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>rotate x:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>move x:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>rotate y:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="rot_x_le">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="move_y_le">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="transform_btn">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>transform</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QComboBox" name="comboBox">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>jpeg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bmp</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>gif</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>type:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="record_btn">
      <property name="text">
       <string>record</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>view/viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
